examples/only-client-render-external-dependencies/components/BarChart.js

```diff
@@ -0,0 +1,12 @@
+import dynamic from 'next/dynamic'
+
+export default dynamic({
+  modules: props => ({
```

Comment:
Lets remove `props` this is going to be remove in Next 7

Reply:
sure. Btw I was not sure how to best approach this example. 
I tried to just make dynamic imports directly in pages like: 
```javascript
const BarChart = dynamic(import('recharts').then(({ BarChart }) => BarChart), { ssr: false }),
const Bar = dynamic(import('recharts').then(({ Bar }) => Bar), { ssr: false })
```
But I experienced charts not being drawn. I think that these two components had to be together ready before they were rendered. The above way solved the issue.

I also thought about just making a directory BarChart and LineChart with `index.js` file exporting:
```javascript
export default dynamic(import('./BarChart'), { ssr: true });
// ... and
export default dynamic(import('./LineChart'), { ssr: true });
```
but this would resolve in `recharts` being bundled twice for these two chunks (and it's quite big).